easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-foss-2015b.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.1.1'
+
+homepage = 'http://ccb.jhu.edu/software/tophat/'
+description = """TopHat is a fast splice junction mapper for RNA-Seq reads."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
+
+patches = [
+    'tophat-2.0.13-zlib.patch',
+]
+
+dependencies = [
+    ('Boost', '1.59.0', '-Python-2.7.10'),
```

Comment:
I'd rather keep the deps the same within the toolchain.

I'll do a foss-2016a version with the bumps

On Wed, Feb 24, 2016 at 3:53 PM Kenneth Hoste <notifications@github.com>
wrote:

> In easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-foss-2015b.eb
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/2567#discussion_r54004473>
> :
>
> > +
> > +homepage = 'http://ccb.jhu.edu/software/tophat/
>
> '
> > +description = """TopHat is a fast splice junction mapper for RNA-Seq reads."""
> > +
> > +toolchain = {'name': 'foss', 'version': '2015b'}
> > +toolchainopts = {'pic': True}
> > +
> > +sources = [SOURCELOWER_TAR_GZ]
> > +source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
> > +
> > +patches = [
> > +    'tophat-2.0.13-zlib.patch',
> > +]
> > +
> > +dependencies = [
> > +    ('Boost', '1.59.0', '-Python-2.7.10'),
>
> Boost 1.60 + Python 2.7.11?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/2567/files#r54004473>
> .
>